vllm_ascend/quant_config.py

```diff
@@ -0,0 +1,179 @@
+"""
+Copyright (c) Huawei Technologies Co., Ltd. 2024-2025. All rights reserved.
+MindIE is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+         http://license.coscl.org.cn/MulanPSL2
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+"""
+
+from typing import Any, Dict, List, Optional
+
+import torch
+import torch_npu
+
+from vllm import _custom_ops as ops
+from vllm.logger import init_logger
+from vllm.model_executor.layers.linear import (LinearBase, LinearMethodBase,
+                                               UnquantizedLinearMethod, RowParallelLinear)
+from vllm.model_executor.layers.quantization import (register_quantization_config)
+from vllm.model_executor.layers.quantization.base_config import (QuantizationConfig)
+from vllm.model_executor.parameter import (BasevLLMParameter,
+                                           GroupQuantScaleParameter,
+                                           PackedvLLMParameter,
+                                           ModelWeightParameter)
+try:
+    from mindie_turbo import get_quantizer, BaseQuantizer
+except:
+    pass
+
+logger = init_logger(__name__)
+
+@register_quantization_config("ascend")
```

Comment:
I wonder if there is a tech name for this quantization method. "ascend" is fine?